src/api-service/__app__/instance_config/__init__.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import azure.functions as func
+from onefuzztypes.enums import ErrorCode
+from onefuzztypes.models import Error
+from onefuzztypes.requests import InstanceConfigUpdate
+
+from ..onefuzzlib.config import InstanceConfig
+from ..onefuzzlib.endpoint_authorization import call_if_user, can_modify_config
+from ..onefuzzlib.events import get_events
+from ..onefuzzlib.request import not_ok, ok, parse_request
+
+
+def get(req: func.HttpRequest) -> func.HttpResponse:
+    return ok(InstanceConfig.fetch())
```

Comment:
Worth explicitly considering: should non-admins be able to query the list of admins? As implemented, they can, right?

Reply:
It was done here on purpose.  I find it exceedingly frustrating when I can't find out who owns a thing I am using.

Reply:
For sure (though "admin" here doesn't imply "owner").

I think this is fine, I just want us to be explicit about our model: we assume all authenticated users (admin or not) can enumerate other users. We don't offer APIs for this purpose, but APIs we provide can be used to do this. Before this PR, it was already possible on some retention horizon, via log queries. Now, it continues to be possible, with this other channel (which discloses admins only). None of this violates any of our existing confidentiality assumptions.